src/python/pants/core/goals/generate_lockfiles.py

```diff
@@ -124,6 +140,156 @@ class RequestedUserResolveNames(Collection[str]):
     """
 
 
+class RequirementVersion(Protocol):
```

Comment:
"Requirement" is Python-specific terminology.

Reply:
it's still not a made up word. Go seems to use "package" and java "artifact" while python use "requirement". Not sure what to use instead..?

Reply:
Yeah dunno, but Requirement is very Python-specific, and will be confusing in any other context.

I think Package is relatively neutral?

Reply:
Yea, agree. Came across package today in another setting as well. 👍🏽